x-pack/plugins/ml/public/timeseriesexplorer/timeseriesexplorer.html

```diff
@@ -115,6 +115,7 @@
           zoom-to="zoomTo"
           auto-zoom-duration="autoZoomDuration">
         </ml-timeseries-chart>
+
```

Comment:
Is this extra line break important ;)

Reply:
Not visible in the diff, the outer `<div>` also has a starting blank line for its children.